src/main/java/me/moodcat/database/embeddables/VAVector.java

```diff
@@ -54,6 +52,9 @@
      *            The arousal of this vector.
      */
     public VAVector(final double valence, final double arousal) {
+        Preconditions.checkArgument(-1 <= valence || valence <= 1);
```

Comment:
These preconditions are always true, no matter what value is inserted. You probably meant &&? This would break the average calculation though, so we would need to find a solution for that.